docs/components/integration-framework/connectors/custom-built-connectors/connector-sdk.md

```diff
@@ -4,17 +4,677 @@ title: Connector SDK
+
+- Fetching and deserializing input data
+- Validating input data
+- Replacing secrets in input data
+
+Beyond that, the SDK allows for [convenient testing](#testing) of your connector behavior and
+[executing it in the environments](#runtime-environments) that suit your use cases best.
+
+## Creating a custom connector
+
+Using the connector SDK, you can create environment-agnostic, reusable connector runtime behavior.
+This section will outline how to set up a connector project, test it, and run it locally.
+
+### Setup
+
+We recommend starting developing a connector using our
+[custom connector template repository on GitHub](https://github.com/camunda/connector-template).
+This template is a [Maven](https://maven.apache.org/)-based Java project, and you can use it the
+way that fits best into your development flow:
+
+- _Create your own GitHub repository_: Click the **Use this template** button and follow the steps.
+  You can manage code changes in your new repository afterward.
+- _Experiment locally_: Check out the source code to your local machine using [Git](https://git-scm.com/).
+  You won't be able to check in code changes to the repository due to restricted write access.
+- _Fetch the source_: Download the source code as a ZIP archive using **Code** > **Download ZIP**.
+  You can adjust and manage the code the way you like afterward, using your chosen source code
+  management tools.
+
+If you want to manualy set up your connector project, include the following dependency to use the SDK.
+Make sure to adhere to the project outline detailed in the next section.
+
+<Tabs groupId="dependency" defaultValue="maven" values={
+[
+{label: 'Maven dependency', value: 'maven' },
+{label: 'Gradle dependency', value: 'gradle' }
+]
+}>
+
+<TabItem value='maven'>
+
+```xml
+<dependency>
+  <groupId>io.camunda.connector</groupId>
+  <artifactId>connector-core</artifactId>
+  <version>0.1.0</version>
+</dependency>
+```
+
+</TabItem>
+
+<TabItem value='gradle'>
+
+```yml
+implementation 'io.camunda.connector:connector-core:0.1.0'
+```
+
+</TabItem>
+</Tabs>
+
+### Project outline
+
+A connector consists of multiple parts that create a reusable building block for modeling and
+runtime behavior. In essence, a connector includes the following things:
+
+```
+my-connector
+├── element-templates/
+│   └── template-connector.json        (1)
+├── src/main
+│   ├── java/io/camunda/connector      (2)
+│   │   ├── MyConnectorFunction.java   (3)
+│   │   ├── MyConnectorRequest.java    (4)
+│   │   └── MyConnectorResult.java     (5)
+│   └── resources/META-INF/services
+│       └── io.camunda.connector.api.ConnectorFunction (6)
+└── pom.xml (7)
+```
+
+Regarding modeling building blocks, the connector provides at least one
+[connector template](./connector-templates.md) with **(1)**.
+
+You provide the runtime logic as Java source code under a directory like **(2)**.
+Usually, a connector runtime logic consists of
```

Comment:
Minor tweak:
Update text from "Usually, a connector runtime logic consists of"
to "Typically a connector runtime logic consists of"

Reply:
Updated